....F....F...........F.F..........F.....F.....
Time: 0.096
There were 6 failures:
1) test37(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test37(RandoopTest0.java:487)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
2) test33(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test33(RandoopTest0.java:439)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
3) test22(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test22(RandoopTest0.java:290)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
4) test21(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test21(RandoopTest0.java:278)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
5) test11(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test11(RandoopTest0.java:135)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
6) test6(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test6(RandoopTest0.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)

FAILURES!!!
Tests run: 40,  Failures: 6,  Errors: 0

Cycle length is4
before
after
Reached this
# of deadlocks detected 0
THE SIZE OF THE TRACE OMEN 34499
Exiting analysis
